app/views/curation_concerns/file_sets/upload/_script_templates.html.erb

```diff
@@ -1,8 +1,7 @@
-<!-- TODO: make the upload_set_id available on the presenter -->
-<% if false && presenter.upload_set_id %>
+<% if @presenter && @presenter.upload_set_id %>
```

Comment:
Then the fix should be to create a presenter when uploading files without a work.